assets/src/room_ui.ts

```diff
@@ -57,6 +93,14 @@ export function setupControls(
     callbacks.onLeave();
     window.location.reload();
   };
+
+
+  simulcastControlsBtn.onclick = () => {
+    state.showSimulcast = !state.showSimulcast
+    setShowSimulcast(state.showSimulcast);
```

Comment:
This button should be visible only when simulcast was enabled so we can do
```suggestion
    changeVisibilitySimulcastControlls(!state.showSimulcast)
```    

and delete `setShowSimulcast`